cmsBuild

```diff
@@ -2520,6 +2535,14 @@ def parseOptions():
                       help="When uploading, copy the temporary directory to the production one.",
                       default=False)
     advancedBuildOptions = OptionGroup(parser, "Advanced build options")
+    advancedBuildOptions.add_option("--source",
+                                    dest="localSources",
+                                    action="append",
+                                    # nargs='+', works only with newer argpars
+                                    # type="str",
```

Comment:
remove the commented out code

Reply:
I think you were checking wrong commit, in this one comments are already removed
https://github.com/cms-sw/pkgtools/pull/141/commits/614bfeee5033056aa041a03e90591ed3b22d07c6